documents/templates/commcomm_to_tsc.md

```diff
@@ -0,0 +1,7 @@
+```
```

Comment:
I don't think it's a realistic barrier, no, because I cannot envision a scenario where someone new to the project would be using these steps. I would even go so far as to say that someone new to this project _shouldn't_ follow these steps. The person following these steps should be the leader of the WG that wants to move, and becoming a leader of a WG isn't going to be someone new to the project (or at least new enough to not be able to file an issue).

Also, this process will be used _very_ infrequently, and whenever we're discussing changes like this it's big. This means there will be multiple TSC and CommComm members available to help with the request.